src/ftxui/dom/text.cpp

```diff
@@ -27,80 +28,93 @@ class Text : public Node {
     int y = box_.y_min;
     if (y > box_.y_max)
       return;
-    for (wchar_t c : text_) {
-      const int width = wchar_width(c);
-      if (width >= 1) {
-        if (x > box_.x_max)
-          return;
-        screen.PixelAt(x, y).character = c;
-      } else {
-        screen.PixelAt(x - 1, y).character += c;
-      }
-      x += std::max(width, 0);
+    for (auto cell : Utf8ToGlyphs(text_)) {
```

Comment:
```suggestion
    for (const auto& cell : Utf8ToGlyphs(text_)) {
```
For consistency with VText::Render.